src/app/shared/module/poe/service/stats/stats.service.ts

```diff
@@ -141,6 +141,57 @@ export class StatsService {
     return results
   }
 
+  public searchExactInType(text: string, statTypesToSearch: StatType[], language?: Language): ItemStat {
+    language = language || this.context.get().language
+
+    let result: ItemStat
+
+    for (const type of statTypesToSearch) {
+      const stats = this.statsProvider.provide(type)
+      for (const tradeId in stats) {
+        if (!stats.hasOwnProperty(tradeId)) {
+          continue
+        }
+
+        const stat = stats[tradeId]
+        const statDescs = stat.text[language]
+        statDescs.forEach((statDesc, statDescIndex) => {
+          if (result) {
+            return
+          }
+          const predicate = Object.getOwnPropertyNames(statDesc)[0]
+          const regex = statDesc[predicate]
+          if (regex.length <= 0) {
```

Comment:
```suggestion
          if (!regex.length) {
```

this should never be possible to be `<0` and 0 is falsey.

Reply:
I've applied your suggestion on this line, and another line (since I kinda copy-pasted this part of the code over here)